code/src/components/Form.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+
+const languages = [' english', ' swedish', ' both'] // array for radio-buttons under language
+
+export const Form = (props) => {
+  const { genre, setGenre } = props
+  const { favorite, setFavorite } = props
+  const { language, setLanguage } = props
+  const { name, setName } = props
+  const { email, setEmail } = props
+
+  return (
+    <div className="form-container">
+      <form>
+        <div className="form-container-sections">
+          <label htmlFor="genre">Pick your genre:</label>
+          <select 
+            id="genre"
+            value={genre}
+            onChange={(e) => {setGenre(e.target.value)}}
+          >
+            <option disabled></option>
+            <option value="biography">Biography</option>
+            <option value="classics">Classics</option>
+            <option value="fiction">Fiction</option>
+            <option value="comedy">Comedy</option>
+            <option value="crime">Crime</option>
+          </select>
+        </div>
+
+        <div className="form-container-sections">
+          <label htmlFor="favorite">The name of your favorite book:</label>
+          <input
+            id="favorite"
+            type="text"
+            placeholder="fav book"
+            value={favorite}
+            onChange={(e) => {setFavorite(e.target.value)}}
+          />
+        </div>
+
+        <div className="form-container-sections">
+          <label htmlFor="language">Select preferred language:</label>
+          {languages.map(lang => (
+            <label key={lang}>
+              <input 
+                id="language"
+                type="radio"
+                value={lang}
+                checked={language === lang}
+                onChange={(e) => {setLanguage(e.target.value)}}
+              />
```

Comment:
Here the id for each input is "language". We should not use the same id for different elements.
Instead htmlFor can be added to the label inside the map.
Then id + htmlFor can be set to {lang}.